proposals/hasCrossSiteAncestor.md

```diff
@@ -104,17 +104,21 @@ string. A UUID of the document.
 
 | Required Combinations | Notes |
 |---|---|
-|frameId | frameId must be a non-zero value|
 |tabId + frameId |frameId can be 0|
 |tabId | will use the top-level frame (frameId 0)|
 |documentId| tabId/frameId are not required but permitted|
 
 ##### Return value
 A Promise that will be fulfilled with a `Cookie.partitionKey` object that matches the properties given in the details parameter and contains the `hasCrossSiteAncestor` value associated with the current cross-site status of the frame.
 
+##### Permissions required to use Cookies.getPartitionKey()
+- `host`
+- `webNavigation`
```

Comment:
Why do we now require the webNavigation permission?

Host permissions on its own already enable extensions to query the origin/partition of a frame, including details about the full ancestor tree (through `location.ancestorOrigins`).

I don't really see a point in requiring webNavigation too.

Reply:
Removed the requirement for webNavigation